posts/v2-metrics.md

```diff
@@ -6,16 +6,16 @@ Two notes. Firstly, the metrics below are roughly ordered by the flow of the sit
 
 ### Visitors signing up (Branding)
 
-One of our aims was to make Exercism feel more welcoming to people who might have been put off by the original (red + devil) branding. To many, this seemed like a minor change, because they **weren't** put off by the branding so the effort seemed like a waste of time. However, we believed that Exercism didn't say "welcoming inclusive community" in V1, and it was important for us to put that right.
+One of our aims was to make Exercism feel more welcoming to people who might have been put off by the original (red + devil) branding. To many, this seemed like a minor change, because they **weren't** put off by the branding and so the effort seemed like a waste of time. However, we believed that Exercism didn't exude "welcoming inclusive community" in V1, and it was important for us to put that right.
 
-So how many people signed up to Exercism over the periods, and what was this as a percentage of new visitors?
+How many people signed up to Exercism over the periods, and what was this as a percentage of new visitors?
 
 V1: 62,618 (17% of new visitors sign up)
 V2: 188,844 (48% of new visitors sign up)
 
-So this first change was substantial. Our V2 landing page converts at 48% as opposed to 17% for V1. That's pretty much purely down to brand and clarity over what we do and who we are. So I strongly believe that the effort that went into rebranding Exercism was worth the effort.
+This first change was substantial. Our V2 landing page converts at 48% as opposed to 17% for V1. That's pretty much purely down to brand and clarity over what we do and who we are. So I strongly believe that the effort that went into rebranding Exercism was worth the effort.
 
-One caveat is that V2 probably encourages people to sign up who aren't yet ready for Exercism as it stands. V2 isn't set up for total beginners, yet the landing page for V2 is much more welcoming for total beginners, so we probably get a lot of users who aren't yet ready for Exercism. One way to account for this would be to say rather than visitors signing up, how many visitors signed up and managed to submit a solution. For this we see:
+One caveat is that V2 probably encourages people to sign up who aren't yet ready for Exercism as it stands. V2 isn't set up for total beginners, yet the landing page for V2 is much more welcoming for total beginners, so we probably get a lot of users who aren't yet ready for Exercism. One way to account for this would be to not show only visitors signing up, but rather how many visitors signed up _and_ managed to submit a solution. For this we see:
```

Comment:
```suggestion
One caveat is that V2 probably encourages people to sign up who aren't yet ready for Exercism as it stands. V2 isn't set up for total beginners, yet the landing page for V2 is much more welcoming for total beginners, so we probably get a lot of users who aren't yet ready for Exercism. One way to account for this would be to consider not just the number of visitors signing up, but rather how many visitors signed up _and_ managed to submit a solution. For this we see:
```